addons/test_event_full/tests/test_event_crm.py

```diff
@@ -148,6 +148,19 @@ def test_event_crm_sale_public(self):
         self.assertEqual(len(self.test_event.registration_ids), self.TICKET1_COUNT + self.TICKET2_COUNT)
         self.assertLeadConvertion(self.test_rule_order, t1_registrations, partner=None)
 
+        # Ensure mails are sent to customers right after subscription
```

Comment:
it's weird to see that test in `test_event_crm`

Reply:
@std-odoo  Indeed, I added a new test, is it ok like this?